### ISSUE TITLE
Improve menu mobile display

### ISSUE THREAD

**Issue by FelixMalfait** [twentyhq/twenty#7826]
https://www.figma.com/design/xt8O9mFeLl46C5InWwoMrN/Twenty?node-id=3122-142184&node-type=frame&t=FDxfW6dnPUgn7ERK-0

## Current behavior
<img width="318" alt="Screenshot 2024-10-18 at 15 15 55" src="https://github.com/user-attachments/assets/86138e3c-d72c-4826-aa51-acefbfbc40d0">

## Desired behavior
![People](https://github.com/user-attachments/assets/8f721584-f04b-4694-b35d-9555b7eb4ef9)

**Comment by FelixMalfait** [comment]
/oss.gg 1200

**Comment by FelixMalfait** [comment]
@ketanMehtaa hey sorry we're not 100% sure we'll go with this design in the end. @Bonapara will confirm soon

**Comment by ketanMehtaa** [comment]
@FelixMalfait okay 
/assign

**Comment by Bonapara** [comment]
Hi @ketanMehtaa edited the desired behavior. We want to use the compact sidebar instead of displaying the menu in a side panel

**Comment by Bonapara** [comment]
The buttons should be 32x32 pixels to make them more easily clickable, as the height is currently only 28 pixels on desktop!

**Comment by Khaan25** [comment]
@Bonapara, Is the bottom menu a slider? Like I've on my portfolio on small devices: https://v2-zzia.vercel.app/

**Comment by FelixMalfait** [comment]
**Note: the compact sidebar already exists (click on the "minize" button on the top-right corner of the left navigation menu), we're not talking about recreating a sidebar from scratch but arranging it so that this vertical sidebar can be displayed horizontally and use as the new navigation bar in mobile mode**

**Comment by AndrewHamal** [comment]
Why is ketan not unassigned its been 2 days already??

**Comment by Khaan25** [comment]
@FelixMalfait, Should I also give it a try?

**Comment by FelixMalfait** [comment]
@Khaan25 Thanks! I think 2 is enough 😄

**Comment by Hitarthsheth07** [comment]
@FelixMalfait On it! Will be done by update the PR by tommorow

**Comment by FelixMalfait** [comment]
Great @Hitarthsheth07 thanks!

**Comment by Hitarthsheth07** [comment]
@FelixMalfait Heyy, 

What should be done with the settings page? Should it be kept the same full screen as it is now?

**Comment by FelixMalfait** [comment]
@Hitarthsheth07 yes that'd be great. Mobile settings don't matter as much, we don't need something that feels native, we just want something that works and doesn't feel clunky :)

**Comment by Hitarthsheth07** [comment]
@FelixMalfait 

I've sorted the navbar issue but cannot figure out a workaround for the settings page. 

So, in the current version when the user clicks on settings page the navbar occupies full screen and the content changes. 
I tried implementing the same thing but it just doesn't work for some reason!

This is the onClick function I am passing to the settings button. 
![image](https://github.com/user-attachments/assets/a9ea965b-0460-4bd3-b667-51149662dc96)

![image](https://github.com/user-attachments/assets/6507b6d0-a460-4c1b-9b79-a02ce2c566c7)

Full Horizontal Navbar
![image](https://github.com/user-attachments/assets/3219ba6d-8956-4465-ac77-ef3e4408f295)

Would love to have suggestions/feedback on it!

Thanks

**Comment by Khaan25** [comment]
Pull the latest code, I with @charlesBochet improved the settings page navigation. I think that might help you out as well. Can't remember the issue number

**Comment by alanstan08** [comment]
Hey if the issue still persists I would love to give it a try

**Comment by lucasbordeau** [comment]
We need to refactor the navigation drawer zone before proceeding with this feature.

Also @Bonapara we should settle on an easy version of this horizontal nav bar, I'll write it down here.
